CHANGELOG.md

```diff
@@ -1,5 +1,87 @@
 # Changelog
 
+## v2.0.0 - 2024-12-02
+
+### Breaking changes
```

Comment:
I would consider moving this section just below the `Highlights` one, so the highlights come first.